deps/spidershim/test/value.cc

```diff
@@ -1434,3 +1434,177 @@ TEST(SpiderShim, ArrayBuffer_NeuteringApi) {
   Local<Float64Array> f64a =
       CreateAndCheck<Float64Array, 8>(buffer, 8, 127);
 }
+
+static inline v8::Local<v8::Script> v8_compile(v8::Local<v8::String> x) {
+  v8::Local<v8::Script> result;
+  if (v8::Script::Compile(v8::Isolate::GetCurrent()->GetCurrentContext(), x)
+          .ToLocal(&result)) {
+    return result;
+  }
+  return v8::Local<v8::Script>();
+}
+
+static inline v8::Local<v8::Value> CompileRun(v8::Local<v8::String> source) {
```

Comment:
On Thu, May 05, 2016 at 11:44:44AM -0700, Ehsan Akhgari wrote:
> In deps/spidershim/test/value.cc:
> 
> > @@ -1434,3 +1434,177 @@ TEST(SpiderShim, ArrayBuffer_NeuteringApi) {
> >    Local<Float64Array> f64a =
> >        CreateAndCheck<Float64Array, 8>(buffer, 8, 127);
> >  }
> > +
> > +static inline v8::Local<v8::Script> v8_compile(v8::Local<v8::String> x) {
> > +  v8::Local<v8::Script> result;
> > +  if (v8::Script::Compile(v8::Isolate::GetCurrent()->GetCurrentContext(), x)
> > +          .ToLocal(&result)) {
> > +    return result;
> > +  }
> > +  return v8::Local<v8::Script>();
> > +}
> > +
> > +static inline v8::Local<v8::Value> CompileRun(v8::Local<v8::String> source) {
> 
> I don't think it will make anything harder, it will make your test consistent
> with the existing ones. I'll do this myself, no point debating this. :-)

The more different our tests are than the v8 ones the more work you need
to do to import them, but if you insist I'll try and remember to do all
the adjustments to use the local style.